### ISSUE TITLE
Testing: Add test for switch link presence in key Lovell pages

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#11995]
Switch links are the canary in the coal mine, if something is wrong with the structure of Lovell.  Switch links may fail to appear or link to the wrong page (linking to itself and not the other version)

## AC's
- Cypress test coverage is present to cover checking for the **appearance** AND the **destination** of the switch link.
 - [x] System health care page Lovell - VA
 - [x] System health care page Lovell - TRICARE
 - [x] Locations page Lovell - VA
 - [x] Locations page Lovell - TRICARE
 - [x] Health Services page Lovell - VA
 - [x] Health Services page Lovell - TRICARE
  - [ ] All Listings Pages 
     - [x] Events Lovell - VA
     - [x] Events Lovell - TRICARE
     - [x] News Releases Lovell - VA
     - [x] News Releases Lovell - TRICARE
     - [x] Stories Lovell - VA
     - [x] Stories Lovell - TRICARE

**Comment by JQuaggles** [comment]
Do not merge until after Lovell Launch

**Comment by swirtSJW** [comment]
This has been merged into the integration branch integration-lovell-tests
